package.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "mei-analyser",
+  "version": "0.2.1-dev",
```

Comment:
I'll make this `0.2.2` and rebase this PR. Everything is at a good state. The only exception is mode detection in Square notation